types/microrouter/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "url-pattern": "^1.0.3"
```

Comment:
do we need this file?

Reply:
We need the URL Pattern import (that's the whole purpose of the PR) and I was following the pattern in other packages.

If I did it wrong, please let me know

Reply:
that's probably fine then! I was wondering why we didn't have one for `micro`